### PR TITLE
Add SFS interfaces

### PR DESCRIPTION
**Pull request by disha-wani**
**What this PR does / why we need it**: This PR adds SFS interfaces

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:NONE

**Release note**:NONE

```release-note
```

### CODE REVIEW

File: openstack/client.go [hunk 1]
```diff
@@ -511,3 +511,10 @@ func NewOBSService(client *golangsdk.ProviderClient, eo golangsdk.EndpointOpts)
 	sc, err := initClientOpts(client, eo, "object")
 	return sc, err
 }
+
+//NewSFSV2 creates a service client taht is used for Huawei cloud  for SFS , it replaces the EVS type.
```
Comment (niuzhenguo) [inline openstack/client.go]: typo, s/taht/that

File: openstack/client.go [hunk 2]
```diff
@@ -511,3 +511,10 @@ func NewOBSService(client *golangsdk.ProviderClient, eo golangsdk.EndpointOpts)
...
+func NewHwSFSV2(client *golangsdk.ProviderClient, eo golangsdk.EndpointOpts) (*golangsdk.ServiceClient, error) {
+	sc, err := initClientOpts(client, eo, "evs")
+	sc.Endpoint = strings.Replace(sc.Endpoint, "evs", "sfs", 1)
```
Comment (niuzhenguo) [inline openstack/client.go]: better to leave a TODO here to use real service type when we add the endpoint for SFS.

File: openstack/client.go (hunk 1 above)
Comment (Savasw) [reply openstack/client.go]: Done

File: openstack/client.go (hunk 2 above)
Comment (Savasw) [reply openstack/client.go]: Comment

Comment (Savasw) [reply openstack/client.go]: Done

**Pull request merged**